modin/pandas/test/test_groupby.py

```diff
@@ -2087,8 +2089,16 @@ def get_columns(df):
     [pytest.param(True, marks=pytest.mark.skip("See modin issue #2513")), False],
 )
 def test_multi_column_groupby_different_partitions(
-    func_to_apply, as_index, by_length, categorical_by
+    func_to_apply, as_index, by_length, categorical_by, request
 ):
+    # breakpoint()
+    if (
+        not categorical_by
+        and by_length == 1
+        and "custom_aggs_at_same_partition" in request.node.name
+        or "renaming_aggs_at_same_partition" in request.node.name
+    ):
+        pytest.xfail("AssertionError: 1.0 >= 0.0001 - float comparison problem")
```

Comment:
add a comment explaining this magic pls; it also feels like a `FIXME` which needs a separate tracker created, too

Reply:
message clarified, tracker - https://github.com/modin-project/modin/issues/6530